bin/hdfcoinc/pycbc_make_psd_estimation_workflow

```diff
@@ -71,74 +71,98 @@ save_fig_with_metadata(time_str, time_file.storage_path, **kwds)
 result_plots += [(time_file,)]
 
 # Get segments and find where the data is
-science_segs, data_segs, science_seg_file = \
-        pycbc.workflow.get_analyzable_segments(workflow, "segments")
-datafind_files, science_segs = pycbc.workflow.setup_datafind_workflow(
-        workflow, science_segs, "datafind", science_seg_file)
-
-cum_veto_files, veto_names, ind_cats = \
-        pycbc.workflow.get_cumulative_veto_group_files(
-                workflow, 'segments-veto-groups', 'segments')
-final_veto_file, final_veto_name, ind_cats = \
-        pycbc.workflow.get_cumulative_veto_group_files(
-                workflow, 'segments-final-veto-group', "segments")
+seg_dir = 'segments'
+
+veto_cat_files = pycbc.workflow.get_files_for_vetoes(
+        workflow, seg_dir, runtime_names=['segments-science-veto'],
+        in_workflow_names=['segments-final-veto-group', 'segments-veto-groups'])
```

Comment:
Do you need "in_workflow_names" here? This implies you are adding jobs to the workflow to run (I assume) CAT 2,3,4 veto generation. I think you only need ['segments-science-veto'] (ie. what is removed in get_analysable_segments). So I think in_workflow_names=None (which is the default value) should be used.